### ISSUE TITLE
Support setting tracklist options in mopidy.conf

### ISSUE THREAD

**Issue by JoshMock** [mopidy/mopidy#2122]
Tracklist options like `mode` , `repeat`, etc. cannot be set in mopidy.conf. They can only be set by a front end that exposes them. My particular use case is that I'd like to set `mode` to `consume` by default on startup, and my preferred front end (mowecl) doesn't expose it, so in order to set it right now, I have to manually make a request to the JSON-RPC API:

```sh
curl \
  -d '{"jsonrpc": "2.0", "id": 1, "method": "core.tracklist.set_consume", "params": [true]}' \
  -H 'Content-Type: application/json' \
  http://localhost:6680/mopidy/rpc
```

**Comment by kingosticks** [comment]
I appreciate this doesn't quite do what you want, but have you seen the restore_state config option? That should re-enable consume mode, but also a load of other stuff so maybe it's not quite what you're looking for.

I'll also mention that it's probably easier to use the mpd frontend rather than the http frontend for simple manual stuff like you're doing.

**Comment by JoshMock** [comment]
Yep, I have `restore_state` set to true in my config, but because state isn't written to disk until service shutdown, if my machine has to be hard-restarted (and I run a Linux distro so this does happen sometimes, especially when waking from hibernation :stuck_out_tongue:) the state file is not written and Mopidy starts back up with empty state.

I guess that'd be another thing to solve for: writing state to disk, on a timer or after each state change&mdash;throttled to avoid excessive writes&mdash;as well as during shutdown, rather than _only_ during shutdown.
